enterprise/ha/src/test/java/org/neo4j/kernel/ha/cluster/SwitchToSlaveTest.java

```diff
@@ -132,7 +134,7 @@ private static SwitchToSlave newSwitchToSlaveSpy()
                 mock( ClusterMemberAvailability.class ), mock( RequestContextFactory.class ),
                 Iterables.<KernelExtensionFactory<?>>empty(), mock( MasterClientResolver.class ),
                 ByteCounterMonitor.NULL, mock( RequestMonitor.class ), mock( SwitchToSlave.Monitor.class ),
-                StoreCopyMonitor.NONE ) );
+                new Monitors().newMonitor( StoreCopyClient.Monitor.class ) ) );
```

Comment:
perhaps you can simply create an empty monitor here: `new StoreCopyClient.Monitor.Adapter()`